src/components/elements/table/GenericTable.tsx

```diff
@@ -87,7 +87,10 @@ export interface Props<T> {
 
 export const clickableRowStyles = {
   backgroundColor: 'background.paper',
-  '&:hover': { backgroundColor: 'grayscale.tint' },
+  '&:hover': {
+    // FIXME(#5746) should be primary[100]
```

Comment:
Maybe this comment should tell us to make it primary[200] to match the hover state in ToolbarMenu? Unless these were intentionally different

Reply:
open Q in figma. I updated the header to use primary.100 for selected state, which is its described purpose. 200 is for hover so I think you're right we should be using that for table, but its dark enough to wash out the borders which looks a little funny.